File: src/main/java/com/github/tanokun/tanorpg/player/quest/action/QuestSendMessageAction.java
package com.github.tanokun.tanorpg.player.quest.action;

import com.github.tanokun.tanorpg.player.Member;
import com.github.tanokun.tanorpg.util.command.CommandContext;
import org.bukkit.Bukkit;
import org.bukkit.entity.Player;

public class QuestSendMessageAction implements Action {

    private String message = "";

    public QuestSendMessageAction(CommandContext cc){
        cc.args.forEach(m -> message = message + " " + m);
    }

    @Override
    public void execute(Member m) {
    Player p = Bukkit.getPlayer(m.getUuid());
    String r = message;
        r = r.replace("[player]", p.getName());
                p.sendMessage(r);
                }
}

File: src/main/java/com/github/tanokun/tanorpg/player/quests/actions/QuestSendMessageAction.kt
package com.github.tanokun.tanorpg.player.quests.actions

import com.github.tanokun.tanorpg.TanoRPG
import com.github.tanokun.tanorpg.player.Member
import com.github.tanokun.tanorpg.player.quests.utils.annotation.ActionData
import com.github.tanokun.tanorpg.util.regex.RegexMatcher

@ActionData("send", "\"<String:text>\"", "send \"[text]\"", "[text]という文字列をplayerに送ります")
class QuestSendMessageAction(regexMatcher: RegexMatcher, parameters: String): QuestAction {
    private var text: String = ""

    init {
        regexMatcher.matchResult(parameters) {
            text = it.get("text", "")
        }
    }

    override fun run(member: Member) {
        var string = text
        string = string.replace("[player]", member.player.name)
        val member = TanoRPG.plugin.memberManager.getMember(member.player.uniqueId)
        if (member != null) string = string.replace("[player_class]", member.skillClass.name)

        member.player.sendMessage(string)
    }
}